docs.html

```diff
@@ -758,6 +759,79 @@ <h3>Demo</h3>
                     </div>
                 </div>
             </section>
+
+            <!-- Grid Component -->
+            <section id="grid" class="component-section">
+                <div class="component-container">
+                    <h2 class="component-heading">Grid</h2>
+                    <span class="component-description">
+                        Grid layout enables us to align elements into columns and rows. 
+                        It excels at dividing a page into major regions or defining the relationship 
+                        in terms of size, position, and layer, between parts of a control built from HTML primitives.
+                    </span>
+
+                    <h3 class="component-variant-heading">Grids with two columns.</h3>
+                    <span>This kind of grid can be created by wrapping the 
+                        elements within <span class="txt-bold">grid-container</span> and <span class="txt-bold">two-col-grid</span> class.</span>
+                    <div class="display-container flex-column-center">
+                        <div class="grid-container two-col-grid">
+                            <div>
+                                <img class="grid-img" src="assets/images/dog-1.png" alt="image of a dog">

```

Comment:
I see that you not applying any grid-related properties in `grid-img`. So it could have been generic to images to be used across the library.

Reply:
Sure, will incorporate this 👍🏽